aerich/migrate.py

```diff
@@ -197,21 +199,23 @@ def _add_operator(cls, operator: str, upgrade=True, fk_m2m_index=False) -> None:
 
     @classmethod
     def _handle_indexes(cls, model: Type[Model], indexes: List[Union[Tuple[str], Index]]) -> list:
-        ret: list = []
-
-        def index_hash(self) -> str:
-            h = hashlib.new("MD5", usedforsecurity=False)  # type:ignore[call-arg]
-            h.update(
-                self.index_name(cls.ddl.schema_generator, model).encode()
-                + self.__class__.__name__.encode()
-            )
-            return h.hexdigest()
-
-        for index in indexes:
-            if isinstance(index, Index):
-                index.__hash__ = index_hash  # type:ignore[method-assign,assignment]
-            ret.append(index)
-        return ret
+        if tortoise.__version__ > "0.22.2":
```

Comment:
When using string comparison, `0.3.1` would be bigger than `0.22.2` but `0.3.1` obviously doesn't have the required methods.

Reply:
Don't worry about that, min tortoise version is 0.11.0 https://github.com/tortoise/tortoise-orm/tags?after=0.11.4

Reply:
> Hey, can you please let me know your arguments for why you want to have the patching code instead of increasing the version requirements?

I think support both tortoise0.21 and tortoise0.22 in aerich-0.8.1 is better than only support 0.22.3+

Reply:
Can you please at least add a comment about why `tortoise.__version__ > "0.22.2"` will do the job? It's very unobvious that generally incorrect code would be producing correct results.